### PR TITLE
[W5.10c][W11-A1] Lai Qi Wei

### PR DESCRIPTION
**Pull request by qwlai**
Added user stories.

### CODE REVIEW

File: doc/DeveloperGuide.md [hunk 1]
```diff
@@ -58,9 +58,11 @@ Priority | As a ... | I want to ... | So that I can...
 `* * *` | new user | see usage instructions | refer to instructions when I forget how to use the App
 `* * *` | user | add a new person |
 `* * *` | user | delete a person | remove entries that I no longer need
+`* * *` | user | edit a person | update the details of a person when the person changes his/her details.
```
Comment (okkhoy) [inline doc/DeveloperGuide.md:61]: You mean edit a person's details :stuck_out_tongue:

**Review by okkhoy — COMMENTED** [review]
I see you have missed the Use cases LO. We are OK as long as you know how to record use cases.

**Comment by okkhoy** [comment]
@qwlai 
Some comments added. Please ack & close the PR after reading comments.

**Comment by qwlai** [comment]
@okkhoy 
Noted with thanks. Apparently, my pull request for use case did not assign to you. The message given by the cs2103-bot was "nus-cs2103-bot assigned hwkchia and unassigned okkhoy 4 days ago".

**Pull request closed without merging**